helm/dagger/templates/engine-statefulset.yaml

```diff
@@ -99,27 +125,63 @@ spec:
           volumeMounts:
             - name: data
               mountPath: /var/lib/dagger
+            {{- if .Values.engine.storage.run.hostPath.enabled }}
             - name: run
               mountPath: /var/run/buildkit
+            {{- end }}
             {{- if .Values.engine.config }}
             - name: config
               mountPath: /etc/dagger/engine.toml
               subPath: engine.toml
             {{- end }}
+            {{- if .Values.engine.configJson }}
+            - name: config
+              mountPath: /etc/dagger/engine.json
+              subPath: engine.json
```

Comment:
What will happen if we have both of these configs @jedevc?

I am wondering whether we should only mount one of these. The fact that we are using the same name for both volume mounts - `config` - will result in an invalid manifest if both are defined. Not that likely to happen, but I can imagine the beginning of a face palm / table flip moment here.

Reply:
While there is a uniqueness constraint on volume names I don't believe there is one on volume mounts since it's not actually a name for the mount, rather a reference to the volume.

Reply:
Ah yes, you are right - this is the reference! 👍